Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:1.13 AS builder
+WORKDIR /go/src/github.com/openshift-kni/cnf-features-deploy
```

Comment:
to speed up the building process can you take a look on this dockerfile 

https://github.com/openshift/linuxptp-daemon/blob/master/Dockerfile#L8

the first part will copy the go.mod file and download all the packages this will cache that layer for the next time we try to build the image

Reply:
there should be no need for downloading anything when vendoring? :thinking:

Reply:
Agree